.github/workflows/dependabot-auto-merge.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       - name: "Merge pull request"
         uses: "actions/github-script@v6"
         with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
+          github-token: "${{ secrets.NESSIE_BUILDER }}"
```

Comment:
maybe add comment why we need a different secret here, compared to the other places in this file